config/vufind/Folio.ini

```diff
@@ -116,6 +116,11 @@ defaultRequiredDate = 0:1:0
 ; "pickUpLocation", "proxiedUsers" and "requestGroup"
 extraHoldFields = requiredByDate:pickUpLocation
 
+; When the extra hold field pickUpLocation is used, this can be used to limit the
+; pickup location (service point) options to those whose assigned locations include the
+; requested item's effective location.
+limitPickupLocations = itemEffectiveLocation
```

Comment:
I did it this way because I can imagine other ways we might want to restrict the pickup location in the future.

Reply:
Should this be commented out by default, or do you think this is desirable enough that it should be enabled by default?

Also, should the comment maybe say something explicitly about item-level requests? (I realize it's fairly clearly implied, but maybe acknowledging the scope would prevent confusion... or maybe it doesn't matter :-) ).

Reply:
Most definitely it should be disabled by default -- thanks, fixed.  And also added that comment...hopefully it's correct :)

Reply:
Looks good to me!